### ISSUE TITLE
Bugclinic scan challenge - wrong problem test ?

### ISSUE THREAD

**Issue by nelson777** [othiym23/bug-clinic#11]
I'm doing bugclinic challanges, but I couldn't find any errors in scan exercice.
The proposed server and client files pass verify with no errors.
Although I have understood the concept of NODE_DEBUG I tought maybe I'm missing something here.

Is this a bug or I'm really missing something ?

**Comment by yoshuawuyts** [comment]
Yeah, same here. Not sure what the test is meant to test, or I'd submit a patch to fix it.

**Comment by alexvicegrab** [comment]
Same here, also, the solution provided in the source does not pass "bug-clinic verify", so there seems to be something off in the challenge.
